applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -165,6 +180,8 @@ KratosGeoMechanicsApplication::KratosGeoMechanicsApplication():
     mUPwUpdatedLagrangianElement2D6N( 0, Element::GeometryType::Pointer( new Triangle2D6 <NodeType >( Element::GeometryType::PointsArrayType(6)))),
     mUPwUpdatedLagrangianElement2D8N( 0, Element::GeometryType::Pointer( new Quadrilateral2D8 <NodeType >( Element::GeometryType::PointsArrayType(8)))),
     mUPwUpdatedLagrangianElement2D9N( 0, Element::GeometryType::Pointer( new Quadrilateral2D9 <NodeType >( Element::GeometryType::PointsArrayType(9)))),
+	mUPwUpdatedLagrangianElement2D10N( 0, Element::GeometryType::Pointer( new Triangle2D10 <NodeType >( Element::GeometryType::PointsArrayType(10)))),
```

Comment:
there are some spaces in the beginning of the line

Reply:
@Vahid-Galavi  Thank you for reviewing this work, very appreciated.

Removed the space. 
These spaces were defined as tabs (by VS IDE) as VS was not showing them.